### ISSUE TITLE
[GridView] Narrator won't announce type of item in GridView

### ISSUE THREAD

**Issue by JordanBluehdorn** [microsoft/microsoft-ui-xaml#9570]
### Describe the bug

When creating a GridView, there is no way to have the narrator describe the type of the component in the reader. This is a problem for vision impaired users who rely on the narrator to help them properly navigate applications.

### Steps to reproduce the bug

1. Create a GridView
2. Make the items of the GridView of type button
3. Open Narrator
4. Return to app and observe

### Expected behavior

Narrator announces the type of the child i.e. `Custom Tile Button 1 of 7`

### Screenshots

Narrator does not announce the type of the child i.e. `Custom Tile 1 of 7`

### NuGet package version

None

**Comment by kmahone** [comment]
You can set AutomationProperties.Name on the elements to give Narrator a description to read out.
